libs/libwebsockets/Makefile

```diff
@@ -61,7 +61,7 @@ endef
 define Package/libwebsockets-mbedtls
 	$(call Package/$(PKG_NAME)/Default)
 	TITLE += (mbedTLS)
-	DEPENDS += +libmbedtls
+	DEPENDS += +libmbedtls @MBEDTLS_VERSION_C
```

Comment:
We should probably activate this option in the main repository by default. Could you create a PR for that too please.